examples/hello_world/config/config.toml

```diff
@@ -1,5 +1,5 @@
 name = "hello_world"
 
 [modules]
-app = { path = "examples/hello_world/bin/hello_world.wasm" }
+app = { path = "examples/hello_world/bin/hello_world.wasm", signature = "" }
```

Comment:
I don't expect the signatures to be part of the application configuration; roughly, I think the application stays as it is, but separately, we provide a list of signatures to the oak_loader, which comes from somewhere else. the idea is that eventually the signatures would come from a verifiable log or anywhere else on the internet, they are not part of the modules nor the application. this would also allow for instance multiple parties to sign the same module, and publish their own signatures in whatever way they prefer.

Reply:
But we need to somehow specify a public key and the signature value for a particular Wasm module

Reply:
Currently I was thinking about this configuration:
```
name = "trusted_information_retrieval"

[modules.app]
location = { path = "examples/trusted_information_retrieval/bin/trusted_information_retrieval.wasm" }
signature = { public_key = "examples/certs/label/label.pub", value = "examples/hello_world/bin/trusted_information_retrieval.wasm.sign" }

[modules.database_proxy]
location = { external = { url = "https://storage.googleapis.com/oak-modules/database_proxy/3606a86d9fa01d223dc3cb092c1aceb9e3afe0ed9799ac5d5b214b1610209f8f", sha256 = "3606a86d9fa01d223dc3cb092c1aceb9e3afe0ed9799ac5d5b214b1610209f8f" } }
```

Reply:
- I think the signature should just be over the hash of the module (maybe in the future we can add additional "tags" or names to a module, but let's ignore the module name for now perhaps).
- a signature would be a (protobuf?) object containing the hash being signed, the public key, and the signature itself.
- the oak_loader is provided with a file (or files, or URLs) of such signatures, and loads them at startup
- when instantiating new modules, the oak runtime looks to see if the module is signed, and by what public key, and verifies the signature against that key (using the minisign format)
- one desired outcome is that if A wants to sign B's module, A can do so without having to re-package B's module; A only needs to publish A's signature over B's module hash

Reply:
Personally I'm leaning towards either having a `.toml` file or packing everything into a config file.